modules/Makefile

```diff
@@ -80,6 +80,7 @@ MODULES_TO_DOCUMENT = \
 	standard/Heap.chpl \
 	standard/Help.chpl \
 	standard/IO.chpl \
+	standard/ChapelIO.chpl \
```

Comment:
This list is otherwise alphabetical, so I'd prefer to put it in the alphabetical place.  I know we're planning on renaming it so it doesn't matter super much, but I don't know how quickly that will happen and would rather we maintain the alphabetization in case the timeframe is long enough that other people adjust this list and fail to recognize it should be alphabetical as a result